server/prisma/schema.prisma

```diff
@@ -12,3 +12,77 @@ datasource db {
   provider = "postgresql"
   url      = env("DATABASE_URL")
 }
+
+model User {
+  zid         String   @id
+  description String?
+  degree      String?
+  year        Int?
+  verified    Boolean
+  admin       Boolean  @default(false)
+  createdAt   DateTime @default(now())
+  updatedAt   DateTime @updatedAt
+
+  reviews Review[]
+}
+
+model Society {
+  id            String   @id @default(uuid())
+  name          String   @unique
+  description   String
+  icon          String   @default("https://kansai-resilience-forum.jp/wp-content/uploads/2019/02/IAFOR-Blank-Avatar-Image-1.jpg")
+  facebookUrl   String?
+  discordUrl    String?
+  emailUrl      String?
+  averageRating Float    @default(0)
+  totalReviews  Int      @default(0)
+  createdAt     DateTime @default(now())
+  updatedAt     DateTime @updatedAt
+
+  reviews Review[]
+  tags    TagsOnSocieties[]
+}
+
+model Review {
+  id        String   @id @default(uuid())
+  title     String?
+  content   String?
+  rating    Int
```

Comment:
for the rating, would it be good to make it a float considering you can give half a star, and would it also be good to enforce this to be between a range e.g 0 - 5

Reply:
I was thinking that for simplicity, we do this by increments of one star (for a review) - similar to how unilectives does this (I think?)

Yep I can do the enforcing - will make the change.